cmd/kluctl/commands/utils.go

```diff
@@ -246,6 +254,30 @@ func parseRepoOverride(s string) (ret repocache.RepoOverride, err error) {
 	if len(sp2) == 3 {
 		ret.Ref = sp2[2]
 	}
-	ret.Override = sp[1]
+	ret.Override = 	os.ExpandEnv(sp[1])
 	return
 }
+
+func parseRepoGroupOverride(s string) (ret repocache.RepoOverride, err error) {
```

Comment:
copy/paste :/ 
could reuse original parseRepoGroup() but the erorr msg would need update. 

for now my intention is to keep it separate, as if use-case would be more then 1:1 replacement it might need different code?

Reply:
Regarding the error message, you can completely remove the `--local-git-override` from the returned error and instead let `withKluctlProjectFromArgs` wrap the returned error messages with the different flag names, e.g.:
```go
ro, err := parseRepoOverride(x, true) // true for --local-git-group-override
if err != nil {
	return fmt.Errorf("invalid --local-git-group-override: %w", err)
}
```